src/main/java/org/springframework/data/neo4j/core/mapping/callback/ReactiveAuditingBeforeBindCallback.java

```diff
@@ -31,7 +33,7 @@
  * @soundtrack Iron Maiden - The Number Of The Beast
  * @since 6.0
  */
-@API(status = API.Status.INTERNAL, since = "6.0")
+@API(status = STABLE, since = "6.0")
 public final class ReactiveAuditingBeforeBindCallback implements ReactiveBeforeBindCallback<Object>, Ordered {
```

Comment:
Do you want to keep that one `public`?

Reply:
Yes. So that people can have a choice either using them directly, via the annotation (`@EnableAuditing…`) or the registrar…). They are different than the id and optimistic locking callbacks as they are not required for a correct function of the whole system.